python/ray/_private/usage/usage_lib.py

```diff
@@ -124,3 +154,81 @@ def get_cluster_metadata(gcs_client, num_retries) -> dict:
             num_retries=num_retries,
         )
     )
+
+
+def generate_report_data(cluster_metadata: dict) -> dict:
+    """Generate the report data.
+
+    Note that we don't perform schema validation.
+
+    The server side is in charge of schema validation, and the schema
+    will be available via public Pypi wheel. The e2e validation will
+    be done from the test.
+
+    cluster_metadata (dict): The cluster metadata of the system generated by
+        `_generate_cluster_metadata`.
+    """
+    data = cluster_metadata.copy()
+    data["collect_timestamp_ms"] = int(time.time() * 1000)
```

Comment:
Please also include the amount of time in seconds since the start of the session, and the sequence number of this report.

Reply:
cc @jjyao this requires the schema change to the backend. Can you take a look at it?

Reply:
@ericl since we already have the collect_timestamp_ms, maybe what we need is just a session_start_timestamp_ms? sequence number I think can just be derived from collect_timestamp_ms since it's ordered.